tests/gradients/core/test_gradient_transform.py

```diff
@@ -198,6 +198,7 @@ class TestGradientTransformIntegration:
     @pytest.mark.parametrize("slicing", [False, True])
     def test_acting_on_qnodes_single_param(self, shots, slicing, atol):
         """Test that a gradient transform acts on QNodes with a single parameter correctly"""
+        np.random.seed(234)
```

Comment:
could we use jax random numbers which are more consistent for testing and reproducability?

Reply:
This is a test added during the gradient restructuring where I forgot to add the seed. It is not a JAX-specific test, so would it be okay to stick to `np.random.seed`? :)